src/test/java/unrc/dose/UserTest.java

```diff
@@ -108,6 +119,71 @@ public void deleteUserUnsuccessful(){
 
 		assertEquals(false, User.disableUser(username, password));
 	}
+	/**
+	 * test that verifies that the password has been modified successfully
+	 * @result false bescause the email is not registered in the DB
+	 */
+	@Test
+	public void updatePasswordUnSuccessfully() {
+		String email = "juanPerez@gmail.com";
+		String pass = "NewPass";
```

Comment:
I do not like this style very much. I think constants should be used directly as parameters in this case.